cobalt/build/configs/BUILD.gn

```diff
@@ -24,16 +24,18 @@ config("cobalt_config") {
     configs += [ "//${starboard_path}/platform_configuration" ]
   } else if (is_cobalt_hermetic_build &&
              current_toolchain == cobalt_toolchain) {
-    assert(current_cpu == "x64",
-           "TODO: b/390500840 - $current_cpu not yet supported.")
     configs += [ "//starboard/build/config:starboard" ]
-    if (sb_is_evergreen) {
-      configs += [ "//starboard/evergreen/x64/platform_configuration" ]
+    if (current_cpu == "arm") {
+      # TODO: b/390500840 - Find a way to switch between hardfp, softfp.
+      if (target_platform == "raspi-2") {
+        configs += [ "//starboard/build/config/modular/arm/hardfp" ]
```

Comment:
This and the below x64 config should be the evergreen configs, not modular configs